gcp/iam.tf

```diff
@@ -9,15 +9,16 @@ resource "google_service_account" "wrongsecrets_workload" {
 
 resource "google_iam_workload_identity_pool" "pool" {
   provider                  = google-beta
-  workload_identity_pool_id = var.project_id
+  workload_identity_pool_id = "other"
```

Comment:
I think only change needs to be reverted. It's clear that it's using the beta provider anyway...